libs/libcommon/include/common/defines.h

```diff
@@ -199,3 +199,9 @@ static ALWAYS_INLINE inline void TIFLASH_NO_OPTIMIZE(T && var)
 #define TIFLASH_DUMMY_FUNCTION_DEFINITION
 #define tiflash_compiler_builtin_memcpy __builtin_memcpy
 #endif
+
+#ifdef __GNUC__
+#define UNUSED_MEMBER
+#else
```

Comment:
I'm not sure if it's better explicit define this macro for `__clang__`.

Reply:
refined and tested in my mac and server.